CMakeLists.txt

```diff
@@ -42,7 +43,6 @@ SET(UTILS_SRC
 )
 
 add_library(utils ${UTILS_SRC})
-
-target_link_libraries(utils ${Boost_SYSTEM_LIBRARY} ${PQ_LIB} log4cplus config ${ZMQ_LIB})
+target_link_libraries(utils ${Boost_REGEX_LIBRARY} ${Boost_THREAD_LIBRARY} config pq zmq log4cplus)
```

Comment:
why remove the library discovery?

Reply:
Because I'm stupid and I can't put back what I was missing around with....